lib/ui/painting/utils.cc

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/common/threads.h"
+#include "flutter/lib/ui/painting/utils.h"
+
+namespace blink {
+
+namespace {
+
+constexpr ftl::TimeDelta kDrainDelay = ftl::TimeDelta::FromMilliseconds(250);
```

Comment:
Rather uncomfortable with ~15 frames worth of delay in collecting items. Can this be significantly lower? Say ~20ms?